spec.bs

```diff
@@ -1192,41 +1194,137 @@ partial interface Navigator {
+    1. If |parsedUpdate["name"] exists and doesn't match |ig|'s [=interest group/name=], 
+      [=iteration/continue=].
+    1. If |parsedUpdate["owner"] exists and doesn't match |ig|'s [=interest group/owner=],
+      [=iteration/continue=].
+    1. [=map/For each=] |key| → |value| of |parsedUpdate|:
+      1. Switch on |key|:
+        <dl class=switch>
+        <dt>"<code>priority</code>"
+        <dd>
+        1. If |value| is a {{double}}, set |ig|'s [=interest group/priority=]
+          to |value|.
+
+        <dt>"<code>enableBiddingSignalsPrioritization</code>"
+        <dd>
+        1. If |value| is a {{boolean}}, set |ig|'s
+          [=interest group/enable bidding signals prioritization=] to |value|.
+
+        <dt>"<code>priorityVector</code>"
+        <dd>
+        1. If |value| is null or an [=ordered map=] whose [=map/keys=] are
+          [=strings=] and whose [=map/values=] are {{double}}, set |ig|'s
+          [=interest group/priority vector=] to |value|.
+
+        <dt>"<code>prioritySignalsOverrides</code>"
+        <dd>
+        1. If |value| is an [=ordered map=] whose [=map/keys=] are
+          [=strings=] and whose [=map/values=] are {{double}} or null:
+          1. [=map/For each=] |pvKey| → |pvValue| of |value|:
+            1. If |pvValue| is null, [=map/remove=] |ig|'s [=interest group/priority signals overrides=][|pvKey|].
+            1. Otherwise, [=map/set=] |ig|'s [=interest group/priority signals overrides=][|pvKey|] to |pvValue|.
+
+        <dt>"<code>executionMode</code>"
+        <dd>
+        1. If |value| is "<code>compatibility</code>" or "<code>group-by-origin</code>",
+          set |ig|'s [=interest group/execution mode=] to |value|.
+
+        <dt>"<code>biddingLogicUrl</code>"
+        <dt>"<code>biddingWasmHelperUrl</code>"
+        <dt>"<code>dailyUpdateUrl</code>"
+        <dt>"<code>trustedBiddingSignalsUrl</code>"
+        <dd>
+        1. For each |groupMember| and |interestGroupField| in the following table <table class="data">
+            <thead><tr><th>Group member</th><th>Interest group field</th></tr></thead>
+            <tr>
+              <td>"biddingLogicUrl"</td>
+              <td>[=interest group/bidding url=]</td>
+            </tr>
+            <tr>
+              <td>"biddingWasmHelperUrl"</td>
+              <td>[=interest group/bidding wasm helper url=]</td>
+            </tr>
+            <tr>
+              <td>"dailyUpdateUrl"</td>
+              <td>[=interest group/daily update url=]</td>
+            </tr>
+            <tr>
+              <td>"trustedBiddingSignalsUrl"</td>
+              <td>[=interest group/trusted bidding signals url=]</td>
+            </tr>
+          </table>
+          1. Let |parsedUrl| be the result of running the [=URL parser=] on |value|.
+          1. [=iteration/continue=] if any of the following conditions hold:
+            * |key| is not |groupMember|;
+            * |parsedUrl| is an error;

```

Comment:
"is failure", since that's what the parser can return. Same with other cases below in this PR.